### DISCUSSION TITLE
How to create a subfolder for each torrent?

### DISCUSSION THREAD

**Discussion by pto199** [BiglySoftware/BiglyBT#3175] [Welcome]
Some torrents you download are packed in folder, but others are not.

My default download directory is  /Downloads.  So all files go in there and it gets cumbersome.  How do I make each torrent added create a sub directory?  I cant find the option to do that.  I know you can do it when manually adding a torrent, but I want it done by default.  Other torrent clients seem to have the option to "create sub directory for each torrent" under where you set the default download directory

Current way Bigly bt saves files from torrents made without folders:

Downloads\File.mp4

How I want it to work:
Downloads\Auto_generated_Folder_Name\File.mp4

**Comment by parg** [comment]
There is no automatic way of doing this.

**Comment by parg** [comment]
Might be able to add an option though, give me a bit

**Comment by parg** [comment]
Join the beta program (see the Help menu) and update to 3501_B27. There's a new option: 

`Options->Files [Default Directory Options]: Always create a sub-folder to contain torrent data files`

that you need to enable.

**Comment by parg** [comment]
I've moved this to an issue, we don't use github discussions.

See https://github.com/BiglySoftware/BiglyBT/issues/3177
